README.md

```diff
@@ -18,9 +18,9 @@ See [Component Documentation](https://sap.github.io/fundamental-react/) for exam
 You will need to install [Node and Node Package Manager](
 https://www.npmjs.com/get-npm).
 
-## Consuming the Library
+## Getting Started
 
-1. Fundamental-react does not include the SAP Fiori Fundamentals library which is required for styling. Install Fiori Fundamentals:
+1. Install Fiori Fundamentals:
```

Comment:
Consumers shouldn't need to do this as it's already a dependency of `fundamental-react`.